lib/gli/monkeypatches/with_indifferent_access.rb

```diff
@@ -0,0 +1,17 @@
+# adapted from https://gist.github.com/Integralist/9503099
+class Object
```

Comment:
Can we not monkeypatch core classes?  This is likely going to break anyone that uses GLI with Rails/ActiveSupport.